src/Mod/PartDesign/Gui/TaskDraftParameters.cpp

```diff
@@ -185,7 +185,7 @@ void TaskDraftParameters::onButtonPlane(bool checked)
         Gui::Selection().clearSelection();
         Gui::Selection().addSelectionGate(new ReferenceSelection(
             this->getBase(),
-            AllowSelection::EDGE | AllowSelection::FACE | AllowSelection::PLANAR));
+            AllowSelection::EDGE | AllowSelection::FACE | AllowSelection::PLANAR | AllowSelection::WHOLE | AllowSelection::SKETCH));

```

Comment:
Can you tell why you also added the WHOLE flag? Doesn't that enable the selection of whole objects that are not Sketches?